ui/widgets/nuxeo-selectivity.js

```diff
@@ -3464,8 +3464,11 @@ typedArrayTags[weakMapTag] = false;
         _updateInputWidth() {
           const inputContent =
             this.input.value || (!this._data.length && this.options.placeholder) || '';
-          this.input.setAttribute('size', inputContent.length + 2);
-
+            if(this._data.length===0 && inputContent.length<=25){
```

Comment:
Rakesh, this solution seems very hacky. Where's the magic number `25` coming from?

Reply:
@richardsd  made it dynamic now using `this.options.placeholder.length`.

thanks for your comment.